packages/contracts-bedrock/SOLIDITY_UPGRADES.md

```diff
@@ -0,0 +1,78 @@
+# Solidity Versioning Policy
+
+This document outlines the process for proposing and implementing Solidity version updates in the
+OP Stack codebase.
+
+## Unified Solidity Version
+
+The OP Stack codebase maintains a single, unified Solidity version across all contracts and
+components. This ensures consistency, simplifies maintenance, and reduces the risk of
+version-related issues.
+
+**Important**: New Solidity versions must not be introduced to any part of the codebase without
+going through the formal version update proposal process outlined in this document.
+
+## Update Process
+
+1. **Minimum Delay Period**: A new Solidity version must be at least 6 months old before it can be
+  considered for adoption.
+2. **Proposal Submission**: Before any Solidity version upgrades are made, a detailed proposal must
+  be submitted as a pull request to the `ethereum-optimism/design-docs` repository in the
+  `solidity/` subfolder, following the standardized format outlined below. This applies to the
```

Comment:
In the design docs repo's `assets` folder we have an FMA template. I'd suggest moving this template there as well